demo-openid/src/Issuer.ts

```diff
@@ -317,12 +317,35 @@ export class Issuer extends BaseAgent<{
       issuerId: '726222ad-7624-4f12-b15b-e08aa7042ffa',
       credentialConfigurationsSupported,
       authorizationServerConfigs: [
+        // TODO: the oid4vc-ts package throws an error if we have authorization servers
+        // for the code auth flow, but then don't use it when using chained authentication.
+        // So for now we comment this out to allow easier testing with Credo as an
+        // authorization server. How to fix it?
+        // {
+        //   type: 'direct',
+        //   issuer: PROVIDER_HOST,
+        //   clientAuthentication: {
+        //     clientId: 'issuer-server',
+        //     clientSecret: 'issuer-server',
+        //   },
+        // },
         {
-          issuer: PROVIDER_HOST,
+          type: 'chained',
+          issuer: 'https://accounts.google.com',
           clientAuthentication: {
-            clientId: 'issuer-server',
-            clientSecret: 'issuer-server',
+            type: 'clientSecret',
+            clientId: 'ID',
+            clientSecret: 'SECRET',
           },
+          supportedCredentials: [
+            {
+              credentialConfigurationId: 'UniversityDegreeCredential-sdjwt',
+              scopes: [
+                'https://www.googleapis.com/auth/userinfo.profile',
+                'https://www.googleapis.com/auth/userinfo.email',
+              ],
+            },
+          ],
```

Comment:
@TimoGlastra we need to think if we want a static configuration like this, or a callback. The callback would allow for more dynamic updates, but it'll move some complexity to the user of Credo. What do you think?